jep/238/README.adoc

```diff
@@ -398,6 +398,16 @@ The format is:
 ]
 ----
 
+=== The description doesn't render well in some cases, what gives?
+
+Jira uses a different markup language to GitHub.
+We're using an HTML export from Jira which does a pretty good job.
+We've also tried Jira markup to Markdown conversion but that had issues as well.
+
+The majority of issues render fine, the ones that don't look good are generally when there's HTML in the Jira issue description, e.g. wrapped around a `noformat` or code block.
```

Comment:
```suggestion
The majority of issues render fine, the ones that don't look good are generally when there's HTML in the Jira issue description, e.g. XML content copied directly in the issue.
```
Ex of such issue: https://github.com/timja-org/jenkins-gh-issues-poc-11-06-a2/issues/2128 vs https://issues.jenkins.io/browse/JENKINS-1918

`{code}`, `{noformat}` and other Jira specific markup fixes are incoming.